tools/test.py

```diff
@@ -52,6 +52,10 @@ def parse_args():
         help='override some settings in the used config, the key-value pair '
         'in xxx=yyy format will be merged into config file. For example, '
         "'--cfg-options model.backbone.depth=18 model.backbone.with_cp=True'")
+    parser.add_argument(
+        '--multi-batches',
```

Comment:
what does multi-batch mean

Reply:
It will determine whether to set `batch` as `cfg.data.videos_per_gpu` first.

Reply:
But `cfg.data.videos_per_gpu` may raise OOM, so this args provide a option for users.

Reply:
i see